rbac/server/api/users.py

```diff
@@ -68,6 +69,41 @@
 
 
 @USERS_BP.get("api/users")
+@doc.summary("Returns all users.")
+@doc.description("Returns all users.")
+@doc.consumes(
```

Comment:
I'm getting this error when I attempt to send the payload in Swagger.
`TypeError: Failed to execute 'fetch' on 'Window': Request with GET/HEAD method cannot have body.`

Reply:
I thought I got all those, must've missed one